.vscode/settings.json

```diff
@@ -9,4 +9,62 @@
     "terminal.integrated.shellArgs.windows": [
```

Comment:
we should look into adding .vscode to the .git_ignore since we don't need these settings in version control